### PR TITLE
Add transfers to engagement_count endpoint

### PR DESCRIPTION
**Pull request by indrekj**
STAT-3

https://github.com/salemove/usage_metrics_recorder/pull/12
https://github.com/salemove/engagement-registry/pull/8
https://github.com/salemove/statistics_service/pull/5
https://github.com/salemove/backend-web/pull/2420
https://github.com/salemove/web_services/pull/770
https://github.com/salemove/apiary_docs/pull/33
https://github.com/salemove/druid-migrations/pull/2

### CODE REVIEW

**Comment by willmore** [comment]
+1

**Comment by stenver** [comment]
+1

File: apiary.apib [hunk 1]
```diff
@@ -2129,7 +2129,7 @@ Returns engagement count.
 Possible dimensions that can be used in filters:
 * `site_id` - Site ID
 * `operator_id` - Operator ID
-* `type` - `proactive`, `reactive`
+* `type` - `proactive`, `reactive`, `transfer`
```
Comment (stenver) [inline apiary.apib]: What if I want `transfer` and `reactive`?

Comment (stenver) [inline apiary.apib]: what type is `transfer` even?

**Pull request merged**